src/containers/ImageUploader/components/ImageMetaData.tsx

```diff
@@ -34,36 +36,48 @@ const ImageMetaData = ({ imageTags, imageLanguage }: Props) => {
           />
         )}
       </FormikField>
-      <FormikField
-        name="modelReleased"
-        label={t("form.modelReleased.label")}
-        description={t("form.modelReleased.description")}
-      >
+      <FormField name="modelReleased">
         {({ field }: { field: FieldInputProps<string> }) => {
           const options = ["yes", "not-applicable", "no", "not-set"];
           const defaultValue = "not-set";
           return (
             <>
-              <RadioButtonGroup
-                selected={field.value ?? defaultValue}
-                uniqeIds
-                options={options.map((value) => ({
-                  title: t(`form.modelReleased.${value}`),
-                  value,
-                }))}
-                onChange={(value: string) =>
-                  field.onChange({
-                    target: {
-                      name: field.name,
-                      value: value,
-                    },
-                  })
-                }
-              />
+              <StyledLabel textStyle="ingress" margin="small">
+                {t("form.modelReleased.label")}
+              </StyledLabel>
```

Comment:
Dette labelet er vel ikke knyttet opp mot noe? Isåfall burde det ikke være et label.

Reply:
Labels kan faktisk også bli implisitt koblet til innhold, altså må man ikke i alle tilfeller definere det eksplititt! Men jeg er enig i at det er overflødig med label her 😅